src/utils/parcel-fetch.ts

```diff
@@ -0,0 +1,23 @@
+export type Parcel= {
+  id: 1
```

Comment:
id is hardcoded to the number 1, maybe better to set it to number.
id: number,

Reply:
Nice catch missed that. Will fix

Reply:
Fixed